packages/store/src/internal/state-factory.ts

```diff
@@ -42,33 +42,31 @@ import { InternalStateOperations } from '../internal/state-operations';
 @Injectable()
 export class StateFactory {
   private _connected = false;
-  private _states: MappedStore[] = [];
-  private _statesByName: StatesByName = {};
 
   constructor(
     private _injector: Injector,
     private _config: NgxsConfig,
     @Optional()
     @SkipSelf()
     private _parentFactory: StateFactory,
+    @Optional() private _internalStateOperations: InternalStateOperations,
```

Comment:
I think that this can be removed as a parameter now.

Reply:
yes!

Reply:
done